LanguageExt.Core/DataTypes/Recursive/Recursive.Prelude.cs

```diff
@@ -0,0 +1,451 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using LanguageExt.DataTypes.Recursive;
+
+namespace LanguageExt
+{
+    public static partial class Prelude
+    {
+
+        /// <summary>
+        /// Returns the base of the recursion
+        /// </summary>
+        /// <typeparam name="A"></typeparam>
+        /// <param name="t"></param>
+        /// <returns></returns>
+        public static Recursive<A> Return<A>(A a) => new Recursive<A>(a);
```

Comment:
I open to suggestions, I just think that `Return` isn't really the right name here.  I think whatever it is should be a single word, so no `RecReturn`, `RecurseReturn` type ugliness.  We should also avoid anything too generic like `Result`.  Thoughts?